src/components/chart-wrapper/index.js

```diff
@@ -32,25 +29,26 @@ const ChartInner = styled('div')`
 
 export const withWrapper = Chart => {
   const ChartWrapper = ({ title, ...chartProps }) => (
-    <Wrapper>
+    <Wrapper chart-title={title}>
       {title.length !==0 && <Title>{title}</Title>}
-      <ChartContainer chart-title={title}>
-        <AutoSizer>
-          {({ height, width }) => (
-            <ChartInner height={height} width={width}>
-              <Chart
-                height={height}
-                width={width}
-                {...chartProps}
-              />
-            </ChartInner>
-          )}
-        </AutoSizer>
-      </ChartContainer>
+      <ResponsiveWrapper>
+        {/* <AutoSizer> */}
+        {({ height, width }) => (
+          // THIS "height - 16" fixes bottom margin, but don't understand know how it works
+          // whithout it the bottom margin doesn't seem to adjust
+          <ChartInner height={height - 16} width={width}>
```

Comment:
This is the weird solution to have the bottom margin adjusting... can set it up maybe to 8...

Reply:
I think this is a strange side effect of a few things:
- wrapping the chart with the `ChartInner` when `ResponsiveWrapper` already wraps in a div
- wrapping the underlying `ResponsiveBar/Scatter/Line/Pie` in another `ResponsiveWrapper`
- the `Title` div taking up extra space, causing the `ResponsiveWrapper` `{ height: 100% }` to overflow
- the `Wrapper` flex styling being at odds with ResizeObserver and/or react-grid-layout

Some weird behaviours:
- changing the value of `height - X` doesn't have much of an effect on the end result. It does seem to mess with how the animations look when **resizing**. e.g. change to `- 5` and it looks the same
- if you remove the `- X` and then tinker with the value of `h` in: https://github.com/EQWorks/chart-system/blob/9ccce2ee57091088cc39cf5d0becd2256691361b/stories/responsive-chart-wrapper.js#L20
  then you can see that the chart only looks messed up when you resize it. For example, keep it at `h: 6` and resize to ~h = 4, you'll see the chart extend past the container. Now change to `h: 4`and reload. The chart will look fine.

I think the following could be a good line:
- convert base charts from `ResponsiveX` to just `X` and then pass `width` and `height`
- remove `ChartInner`
- remove flex styling, which seemed to disrupt the resizing
- pass `height={height - adjustForTitle}`

did it here:

Reply:
Thanks Shane! I did try removing the ChartInner and have a non-responsive component, but didn't find your solution to be able to include the component... I will incorporate this now...

Reply:
The only problem we have to solve here, is when the title component upon shrinking wraps on the next line...

Reply:
getting the height from `ref` works!

Reply:
Yes, I used ref!